packages/wasm-parser/test/fixtures/import/complex/actual.wat

```diff
@@ -0,0 +1,5 @@
+(module
```

Comment:
this simple test, which adds a couple of type definitions, was enough to repro the issues from #406